packages/test-interface-criteria/src/index.js

```diff
@@ -4,12 +4,54 @@
  */
 const criteria = require("./criteria").criteria.map(x => new x());
 
+function getOffers(opportunity) {
+  return opportunity.offers || (opportunity.superEvent && opportunity.superEvent.offers) || []; // Note FacilityUse does not have bookable offers, as it does not allow inheritance
+}
+
+function filterRelevantOffers(criteria, opportunity) {
+  const offers = getOffers(opportunity);
+  return Object.entries(criteria.offerConstraints)
+    .reduce((offers, [, test]) => offers.filter(offer => test(offer, opportunity)), offers);
+}
+
+function testMatch(criteria, opportunity) {
```

Comment:
I would personally benefit from a lil doc string for this function as I think it's rather important. It takes me a while to understand what this function does just by reading it

Reply:
Actually nvm. I was just confused because `criteria` is the name of the param as well as the module-level variable (as mentioned in another comment)

Reply:
Ah sure, though can easily add a doc string!